cs/neon.texy

```diff
@@ -22,58 +22,58 @@ use Nette\Neon\Neon;
 encode(mixed $value, int $flags = 0): string .[method]
 ------------------------------------------------------
 
-Returns `$value` encoded into NEON. Flags accepts `Neon::BLOCK` which formats NEON in multiline format.
+Vrací `$value` převedenou do NEONu. Jako druhý parametr můžeme použít `Neon::BLOCK`, který vytvoří víceřádkový výstup.
 
 /--php
-Neon::encode($value); // Returns $value encoded in NEON
-Neon::encode($value, Neon::BLOCK); // Returns formatted $value encoded in NEON
+Neon::encode($value); // Vrátí $value převedenou do NEONu
+Neon::encode($value, Neon::BLOCK); // Vrátí $value převedenou do víceřádkového NEONu
 \--
 
-Method `encode()` throws `Nette\Neon\Exception` on error.
+Metoda `encode()` při chybě vyhodí `Nette\Neon\Exception`.
 
 /--php
 try {
 	$neon = Neon::encode($value);
 } catch (Nette\Neon\Exception $e) {
-	// Exception handling
+	// zpracování výjimky
 }
 \--
 
 
 decode(string $neon): mixed .[method]
 -------------------------------------
 
-Converts given NEON to PHP value.
+Převede vstup z NEONu do PHP.
 
 /--php
-Neon::decode('hello: world'); // Returns an array ['hello' => 'world']
+Neon::decode('hello: world'); // Vrátí pole ['hello' => 'world']
 \--
 
-Method `decode()` throws `Nette\Neon\Exception` on error.
+Metoda `decode()` při chybě vyhodí `Nette\Neon\Exception`.
 
 /--php
 try {
 	$value = Neon::decode($neon);
 } catch (Nette\Neon\Exception $e) {
-	// Exception handling
+	// zpracování výjimky
 }
 \--
 
 
-See [NEON for JavaScript |https://github.com/matej21/neon-js] and [NEON for Python |https://github.com/paveldedik/neon-py].
+Viz také [NEON for JavaScript |https://github.com/matej21/neon-js] a [NEON for Python |https://github.com/paveldedik/neon-py].
 
 
-IDE integration
-===============
+Integrace do IDE
+================
 
-NetBeans IDE has built-in support, check out [NEON plugin for PhpStorm |https://plugins.jetbrains.com/plugin/7060?pr]
-and [Visual Studio Code |https://marketplace.visualstudio.com/items?itemName=Kasik96.latte].
+NetBeans má podporu vestavěnou, pro PhpStom existuje [NEON plugin |https://plugins.jetbrains.com/plugin/7060?pr] a pro Visual Studio Code 
+[taktéž |https://marketplace.visualstudio.com/items?itemName=Kasik96.latte].
 
 
-Syntax
-======
+Syntaxe
+=======
 
-NEON is superset of JSON. So every JSON can be parsed as NEON:
+NEON je nadmnožinou JSONu. Každý JSON se dá zparsovat jako NEON:
```

Comment:
NEON je nadmnožinou. Co je validní JSON, je automaticky i validní NEON. Co je validní NEON, to nemusí být validní JSON. Dá se to i otočit a říct, že JSON je podmnožina NEONu.